docs/reference/migration/migrate_5_0/suggest.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[breaking_50_suggester]]
+=== Suggester changes
+
+Now `completion` and `context` suggest fields are indexed as lucene's
+suggest fields. This means new `completion` indices are not compatible
+with pre-5.0 indices. Older `completion` indices can still be queried
+and indexed in the same way as before but not created. It is strongly
+recommended to reindex `completion` fields to take advantage of the
+following:
```

Comment:
I'd replace this para with:

    The completion suggester has undergone a complete rewrite. This means that the
    syntax and data structure for fields of type `completion` have changed, as
    have the syntax and response of completion suggester requests.

    For indices created before Elasticsearch 5.0.0, `completion` fields and the
    completion suggester will continue to work as they did in Elasticsearch 2.x.
    However, it is not possible to run a completion suggester query across indices
    created in 2.x and indices created in 5.x.

    It is strongly recommended to reindex indices containing 2.x `completion`
    fields in 5.x to take advantage of the new features listed below.

    NOTE: You will need to change the structure of the completion field values
    when reindexing.